### DISCUSSION TITLE
Update Triggers

### DISCUSSION THREAD

**Discussion by TheITCloudGuy** [Weatherlights/Winget-AutoUpdate-Intune#29] [General]
Hi there,

I was wondering how the triggers work with this project now that it has moved to the Store Installation? I previously had WAU running from Intune via the win32 app, but migrated over to the Store App for ease of use. But alot of our devices are displaying the Schtasks for logon as previously used, am I missing something in the deployment?

These are the Tasks I have, there is nothing under the Main Task Scheduler Library.
![image](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/assets/43003164/45a84bd1-aa6f-40b2-950b-aa53f13593ba)

**Comment by Weatherlights** [comment]
The configuration tasks configures Winget Autoupdate. So after the task has run the tasks from WAU should show up.

The configuration task gets fired when an Intune synchronisation was fired.
